pkg/quarkus/v1alpha/scaffolds/api.go

```diff
@@ -77,6 +78,10 @@ func (s *apiScaffolder) Scaffold() error {
 			Package:   util.ReverseDomain(util.SanitizeDomain(s.config.GetDomain())),
 			ClassName: util.ToClassname(s.resource.Kind),
 		},
+		&templates.Makefile{
+			Image:            "",
+			KustomizeVersion: "v3.5.4",
+		},
```

Comment:
Why is this only part of `create api` instead of being part of `init`? I think it needs to be part of `init` and if there is something you need that is only available from `create api` we need to create a makefile fragment that can be added to the `Makefile` when `create api` is run.

Reply:
Yes, I need to create Makefile in `inti` and then add fragments during `create api`. I will update the PR accordingly.

Reply:
@jmrodri - I tried to use fragment logic in an same way from Helm Operator scaffolding for `watches.yaml` (https://github.com/operator-framework/operator-sdk/blob/master/internal/plugins/helm/v1/scaffolds/internal/templates/watches.go) But, failing with below error as it only supports go, yaml, and yml file extensions.

[
<img width="1534" alt="Screen Shot 2022-05-23 at 1 05 16 PM" src="https://user-images.githubusercontent.com/47066536/169899694-825828de-ad75-46af-b23b-7850e7a3a4c3.png">
](URL)

Looking for other options like reading the file and then writing to it.

Reply:
@laxmikantbpandhare could you use something like the Kubebuilder utils `InsertCode` function that we normally use during testdata generation?  Something like:
https://github.com/operator-framework/operator-sdk/blob/b7efd5241623d72eacb01ce192c6d19cff1d10ca/hack/generate/samples/internal/go/v3/memcached_with_webhooks.go#L271-L279

Reply:
Sure @everettraven. I will go through it. Right now, looking into PostScaffold method.